Project/src/main/java/com/github/lgooddatepicker/components/CalendarPanel.java

```diff
@@ -300,6 +300,15 @@ public void integerTextFieldNumberChanged(JIntegerTextField source, int newValue
                 drawCalendar(newYearMonth);
             }
         };
+        // Code passes Maven's tests and compiles successfully.
```

Comment:
> Code passes Maven's tests and compiles successfully.

This comment is more fitting for your *pull request* description. It should not part of the library code. Please remove it.